spec/rspec/features_spec.rb

```diff
@@ -70,8 +70,18 @@ def current_example(context)
   end
 end
 
-feature "if xscenario aliases to pending then" do
-  xscenario "this test should be 'temporarily disabled with xscenario'" do
+xfeature "if xfeature aliases to pending then" do
```

Comment:
This needs a condition so the test is not run for RSpec < 3  - probably the other additions too